workers/Dockerfile

```diff
@@ -1,5 +1,13 @@
+# based on Debian
 FROM python:3.8
 
+# install R
+# to avoid caching issues combine apt-get update and install in one RUN statement.
+# to reduce image size, clean up the apt cache by removing /var/lib/apt/lists.
+RUN apt-get update && \
+    apt-get install -y r-base && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
I don't this this is sufficient, please have a look at the [CI results](https://jenkins.ohsome.org/blue/organizations/jenkins/ohsome-quality-analyst/detail/PR-170/109/pipeline).